app/SentryServiceProvider.php

```diff
@@ -26,20 +25,19 @@ function (Application $app) {
             }
         );
 
-        $app['uncaught_error_handler'] = $app->share(
+        $app[SentryErrorHandler::class] = $app->share(
             function ($app) {
-                /** @var HubInterface $sentryHub */
-                $sentryHub = $app[HubInterface::class];
-
-                return static function (Throwable $throwable) use ($sentryHub) {
-                    $sentryHub->captureException($throwable);
-                    throw $throwable;
-                };
+                return new SentryErrorHandler(
+                    $app[HubInterface::class],
+                    $app['jwt'] ?? null,
+                    $app['auth.api_key'] ?? null,
+                    $app['api_name'] ?? null
```

Comment:
This parameter is not nullable according to https://github.com/cultuurnet/udb3-silex/pull/507/files#diff-64fd9397ae1ed3a3e08bd0a9da5c7f02R27 so the default should be an empty string instead. Or make it nullable.

Reply:
But good idea to also include it! 👍

Reply:
I made it consistent with other parameters and it's also nullable now, see https://github.com/cultuurnet/udb3-silex/pull/508